integration/spark2/src/main/scala/org/apache/carbondata/spark/util/CarbonSparkUtil.scala

```diff
@@ -51,4 +55,90 @@ object CarbonSparkUtil {
       table)
   }
 
+  /**
+   * Below method will be used to get the fields object for pre aggregate table
+   *
+   * @param columnName
+   * @param dataType
+   * @param aggregateType
+   * @param parentColumnId
+   * @param parentTableName
+   * @param parentDatabaseName
+   * @param parentTableId
+   * @return fields object
+   */
+  def getField(columnName: String,
```

Comment:
Why do you require this code? you ultimately require `rawSchema` , that you can anyway get from `dataType.catalogString`, just append the comment to it and use in createtablecommand